docs/manual/hacking/guidelines.md

```diff
@@ -1,71 +1,84 @@
 # Guidelines {#sec-guidelines}
 
-If your contribution tightly follows the guidelines, then there is a good chance it will be merged without too much
-trouble. Some of the guidelines will be strictly enforced, others will remain as gentle nudges towards the correct
-direction. As we have no automated system enforcing those guidelines, please try to double check your changes before
-making your pull request in order to avoid "faulty" code slipping by.
+If your contribution tightly follows the guidelines, then there is a good chance
+it will be merged without too much trouble. Some of the guidelines will be
+strictly enforced, others will remain as gentle nudges towards the correct
+direction. As we have no automated system enforcing those guidelines, please
+try to double check your changes before making your pull request in order to
+avoid "faulty" code slipping by.
 
-If you are uncertain how these rules affect the change you would like to make then feel free to start a
-discussion in the [discussions tab](https://github.com/NotAShelf/neovim-flake/discussions) ideally (but not necessarily)
-before you start developing.
+If you are uncertain how these rules affect the change you would like to make
+then feel free to start a discussion in the [discussions tab](https://github.com/NotAShelf/nvf/discussions)
+ideally (but not necessarily) before you start developing.
 
 ## Adding Documentation {#sec-guidelines-documentation}
 
-Most, if not all, changes warrant changes to the documentation. Module options should be documented with
-[Nixpkgs-flavoured Markdown](https://nixos.org/manual/nixpkgs/unstable/#sec-contributing-markup), albeit with exceptions.
-
-:::{.note}
-
-As of v0.5, neovim-flake is itself documented using full markdown in both module options and the manual. With
-v0.6, this manual has also been converted to markdown in full.
+Most, if not all, changes warrant changes to the documentation. Module options
+should be documented with [Nixpkgs-flavoured Markdown](https://nixos.org/manual/nixpkgs/unstable/#sec-contributing-markup),
+albeit with exceptions.
 
+::: {.note}
+As of **v0.5**, **vf** is itself documented using full markdown in both module
```

Comment:
vf?

Reply:
```suggestion
As of **v0.5**, **nvf** is itself documented using full markdown in both module
```

Reply:
I accidentally leaked the future future name...